es-bs-base/scss/vue/components/_form-rating.scss

```diff
@@ -57,3 +57,9 @@
     }
   }
 }
+
+.rating {
+    display: flex;
+    height: 4.5rem;
+    width: 9.8rem;
```

Comment:
It looks like this class is only used by the Rating docs page - if so, it shouldn't be part of the `es-bs-base` package, it should be a style in a `<style>` tag on that Rating docs page.

Also - you can essentially accomplish all of this using utility classes `d-flex p-200` on the div elements on the docs page itself, so you shouldn't need a `<style>` tag at all.

Reply:
Thanks for the feedback! I have updated the code as suggested.